packages/web-app-admin-settings/src/views/Spaces.vue

```diff
@@ -182,13 +183,24 @@ export default defineComponent({
         {
           app: 'SpaceMembers',
           icon: 'group',
-          title: $gettext('Space members'),
+          title: $gettext('Members'),
           component: MembersPanel,
           default: false,
           enabled: unref(selectedSpaces).length === 1,
           componentAttrs: {
             spaceResource: unref(selectedSpaces)[0]
           }
+        },
+        {
+          app: 'SpaceActions',
```

Comment:
Please insert the `Actions` panel before the `Members` panel. Otherwise the order of panels is inconsistent in the files app (project spaces overview) and here in the admin settings app.

Reply:
(took care myself, so that we can merge this asap)